GiniVision/Classes/CameraViewController.swift

```diff
@@ -17,13 +17,28 @@ import AVFoundation
 public typealias CameraSuccessBlock = (_ imageData: Data) -> ()
 
 /**
- Block which will be executed if an error occurs on the camera screen. It contains a camera specific error.
+ Block which will be executed when the camera screen successfully takes a picture or pick a document/picture. It contains the JPEG representation of the image including meta information about the image, or the PDF Data. Should be indicated if the document was imported or not.
+ 
+ - note: Component API only.
+ */
+public typealias CameraScreenSuccessBlock = (_ document: GiniVisionDocument, _ isImported:Bool) -> ()
```

Comment:
Why would clients care if it was imported or not? I'm not sure I agree with that...

Reply:
In order to keep the same methods and avoid companies to make any change to their current implementation, we added the new delegate method `didImport`. We use that property when we switch between calling that or method or `didCapture`.